examples/python/requirements.txt

```
@@ -0,0 +1,3 @@
+PyJWT
+httpx
+cryptography
```

One thing we could do here is pin the dependencies just to make sure a future upgrade doesn't break things on us:
```suggestion
PyJWT==2.9.0
httpx==0.27.2
cryptography==43.0.1
```